test/BF16/vnni-to-tpp.mlir

```diff
@@ -0,0 +1,33 @@
+// RUN: tpp-opt  --one-shot-bufferize="bufferize-function-boundaries allow-return-allocs function-boundary-type-conversion=identity-layout-map" --transform-dialect-interpreter %s | FileCheck %s 
+
+module {
+  transform.sequence failures(propagate) {
+  ^bb0(%arg0: !pdl.operation):
+    %0 = transform.structured.match ops{["matmul_static"]} in %arg0
+    transform.structured.map_vnni_to_tpp %0
+  }
+
+  transform.sequence failures(propagate) {
```

Comment:
Why the two ways? Don't they mean the same thing? Which one applies first?

Reply:
my bad, I'll get rid of the redundant one.